Docs/Differences/Differences.tex

```diff
@@ -2454,6 +2454,17 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
   \emph{Note}: This option is a substitution for BT4LEContinuityFixup.kext,
   which does not function properly due to late patching progress.
 
+\item
```

Comment:
This file is generated automatically when BuildDocs.tool is executed. Do not edit it. We can build the documentation ourselves, so please limit your changes to Configuration.tex.

Reply:
Done